internal/usage/usage.go

```diff
@@ -269,7 +272,8 @@ dbName and collection are required only for built-in roles.`
 	SubscriptionID                            = "Unique identifier of the Azure subscription in which the VNet resides."
 	GCPProjectID                              = "Unique identifier of the GCP project in which the network peer resides."
 	Network                                   = "Unique identifier of the Network Peering connection in the Atlas project."
-	APIRegion                                 = "Indicates which API URL to use, either US or EU. The integration service will use US by default."
+	APIRegion                                 = "Code that indicates which regional URL MongoDB uses to access the third-party API. Valid values are US and EU."
+	DatadogAPIRegion                          = "Code that indicates which regional URL MongoDB uses to access the Datadog API. Valid values are US, EU, US3, and US5."
```

Comment:
Suggestion:
```suggestion
	APIRegion                                 = "Two-letter country dode that indicates which regional URL MongoDB uses to access the third-party API. Valid values are US and EU."
	DatadogAPIRegion                          = "Two-letter country code that indicates which regional URL MongoDB uses to access the Datadog API. Valid values are US, EU, US3, and US5."
```

Reply:
Oops... please ignore the above comment. Just noticed that some of them aren't two letters :-) Please ignore previous comment. Thanks!